samples/word/50-document/manage-settings.yaml

```diff
@@ -0,0 +1,133 @@
+order: 6
+id: word-document-manage-settings
+name: Manage settings
+description: 'This sample shows how to add, get, and delete settings in the file.'
+host: WORD
+api_set:
+    WordApi: '1.4'
+script:
+    content: |
+        $("#create-setting").click(() => tryCatch(createSetting));
+        $("#get-settings").click(() => tryCatch(getSettings));
+        $("#delete-all-settings").click(() => tryCatch(deleteAllSettings));
+
+        async function createSetting() {
+          await Word.run(async (context) => {
+            let key = $("#key")
```

Comment:
Let's add some "thesis statement" comments to the starts of each of these methods. They'll help the reader both here and in the reference docs once these snippets are plucked over.

Reply:
Might be worth noting for the add that adding a setting to an existing key replaces the existing value.